app/utils/string_test.go

```diff
@@ -1,6 +1,8 @@
 package utils
```

Comment:
A package cannot be called 'utils' as all the packages should have meaningful names. 'utils' is a classic example of bad package names. We should do something with it (see https://go.dev/blog/package-names)